### ISSUE TITLE
Problems with crop page sidebar display

### ISSUE THREAD

**Issue by Skud** [Growstuff/growstuff#633]
The display of plantings and harvests in the crop detail page sidebar is a bit off.  If some details aren't filled in, it comes out as "Skud planted ." or similar.

![screenshot 2015-01-15 10 34 50](https://cloud.githubusercontent.com/assets/103436/5750068/61525e0e-9ca3-11e4-84de-9a3f91ce84a0.png)

This seems to be a result of [this work](https://github.com/Growstuff/growstuff/commit/14d039ee018b4d877df10c17285d7a0f471fecd4) which means the problem's been there for a few months.  However I think it's less likely to show up on production (the screenshot is from staging) because people tend to fill in more details there.  Here's a screenshot from production:

![screenshot 2015-01-15 10 44 59](https://cloud.githubusercontent.com/assets/103436/5750079/99494d2c-9ca3-11e4-8329-e89ea0421eab.png)

I've marked it as "severity: major" because although happens less on production, the crop detail page is our showcase, and this is definitely higher priority than most of the "severity: minor" bugs.

I would suggest looking at Harvest.to_s and seeing whether we can use that for harvests at least, and whether we can do something similar for plantings, to standardise this kind of display.

**Comment by pozorvlak** [comment]
Also, "3 advanced plant" should be pluralized.

**Comment by gabrielle27** [comment]
I just submitted a pr for the plantings portion of this.  If that looks ok, I could do the same for harvests.  Here's a screenshot of the possible display types.  I temporarily expanded my sandbox to display 4 at a time so I could capture all of them.

There might be some better suggestions for language.  
- If planted_from is missing I default to 'units'.
- If both planted_from and quantity are missing I just included the member name, which is a bit terse, but answers the question of "who's planting?"
- If quantity is missing I just include planted_from.
- Last item on the screen shot is normal, and shows the addition of pluralization.


![screen shot 2015-02-07 at 8 45 38 am](https://cloud.githubusercontent.com/assets/416920/6092899/49f6f560-aea7-11e4-9bc0-7b099508d034.png)
